kivy/resources.py

```diff
@@ -42,23 +43,42 @@
     resource_paths += [join(dirname(sys.argv[0]), 'YourApp')]
 resource_paths += [dirname(kivy.__file__), join(kivy_data_dir, '..')]
 
+Cache.register('kv.resourcefind', timeout=60)
 
-def resource_find(filename):
+
+def resource_find(filename, use_cache=True):
```

Comment:
```suggestion
def resource_find(filename, use_cache=environ.get('KIVY_DOC_INCLUDE', None) != '1'):
```

Reply:
added suggestions in a way to mirror clock and imported os.environ rather than os.path.environ

Reply:
Technically, the way clock does it is wrong, because we need to check the actual value, not just that it has been defined. E.g. someone could have `KIVY_DOC_INCLUDE=0` rather than deleting the variable from their env. But it's fine since hopefully we'll remove it anyway in the not so distant future.